libraries/chain/services/account.cpp

```diff
@@ -365,6 +365,12 @@ void dbs_account::update_voting_power(const account_object& account, uint16_t vo
     time_point_sec t = db_impl().head_block_time();
 
     update(account, [&](account_object& a) {
+        if (voting_power < a.voting_power && a.active_sp_holders_cashout_time == fc::time_point_sec::maximum())
+        {
+            fc::time_point_sec cashout_time = t;
+            cashout_time += SCORUM_PRODUCER_REWARD_PERIOD;
```

Comment:
NOTE:
rename `SCORUM_PRODUCER_REWARD_PERIOD` to `SCORUM_ACTIVE_HOLDER_REWARD_PERIOD`